wasm/Cargo.toml

```diff
@@ -18,7 +18,7 @@ path = "src/mod.rs"
 
 [dependencies]
 clvm_tools_rs = { path= "..", features = [] }
-clvmr = "0.1.21"
+clvmr = "0.1.24"
```

Comment:
we even have `0.2.0` now (`0.2.1` has a bug that I'm working on fixing)

Reply:
mainly haven't needed an update, but will look into it.